.travis.yml

```diff
@@ -33,31 +33,45 @@ env:
 
 matrix:
   include:
-    #bitcoind clang (no depend, only system lib installed via apt)
-    - compiler: clang
+    #x86_64 Linux + deps as via system lib
+    - compiler: gcc
       env:
-        - CXX=clang++-5.0 CC=clang-5.0
-        - CXXFLAGS="-std=c++14"
+        - A="ia64 linux  full tests  system deps, enable debug, debug boost"
```

Comment:
This doc shows how to name jobs, but we are using some different feature  "matrix".  The linked docs mention matrix but not how to name them.  Do you know how?

Reply:
one thing I like about putting them as env vars is that they show up in the travis synopsis page in a field that's not useful to us because we have so many environment vars that the field just shows the same thing unless you roll over it.

Reply:
> This doc shows how to name jobs, but we are using some different feature "matrix". The linked docs mention matrix but not how to name them. Do you know how?

I think that `jobs` and `matrix` are different declination of the same concept and that `name` works in both of them.

That said we could leave it for another PR.

Reply:
Let's do that so we can get these fixes deployed to the other PRs so the don't fail spuriously as much.